docker-compose.yml

```diff
@@ -45,6 +45,7 @@ services:
       EASY_DEV_MODE: "yes"
       EASY_DEV_MODE_NEW: "yes"
       DEVELOPER_TOOLS: "yes"
+      TURN_API_ON: "yes"
```

Comment:
change this to ACTIVATE_API
(gonna bring this in right after this is fixed to get ready for the new dockers :) )

Reply:
Done @bradymiller